README.md

```diff
@@ -28,13 +28,14 @@ You can read more about GeckoView on the [wiki](https://wiki.mozilla.org/Mobile/
 
 
 [1]:https://geckoview.dev/consumer/docs/geckoview-quick-start
-[2]:https://geckoview.dev/javadoc/mozilla-central/org/mozilla/geckoview/doc-files/CHANGELOG.md
-[3]:https://geckoview.dev/contributor/geckoview-quick-start.md
-[4]:https://geckoview.dev/contributor/mc-quick-start.md
-[5]:https://geckoview.dev/contributor/contributing-to-mc.md
-[6]:https://geckoview.dev/contributor/native-debugging.md
-[7]:https://geckoview.dev/consumer/docs/web-extensions.md
+[2]:https://mozilla.github.io/geckoview/javadoc/mozilla-central/org/mozilla/geckoview/doc-files/CHANGELOG
```

Comment:
these should still link to `geckoview.dev`:  `https://geckoview.dev/javadoc/mozilla-central/org/mozilla/geckoview/doc-files/CHANGELOG`

(applies to all of them)